mods/cnc/maps/nod04b/nod04b.lua

```diff
@@ -0,0 +1,184 @@
+Civ1CellTriggerActivator = { CPos.New(24,52), CPos.New(23,52), CPos.New(22,52), CPos.New(23,51), CPos.New(22,51), CPos.New(21,51) }
+Civ2CellTriggerActivator = { CPos.New(26,54), CPos.New(25,54), CPos.New(24,54), CPos.New(25,53), CPos.New(24,53), CPos.New(23,53) }
+
+Apc1Units = { 'c2', 'c3', 'c4', 'c5' }
+
+WinActorTriggerActivator = { Civilian1, Civilian2, Civilian3, Civilian4, Civilian5, Civilian6, Civilian7, Civilian8, CivBuilding1, CivBuilding2, CivBuilding3, CivBuilding4, CivBuilding5, CivBuilding6, CivBuilding7, CivBuilding8, CivBuilding9, CivBuilding10, CivBuilding11, CivBuilding12, CivBuilding13, CivBuilding14 }
+Apc2ActorTriggerActivator = { NodGunner1, NodGunner2, NodGunner3 }
+
+Apc1Waypoints = { waypoint0.Location, waypoint11.Location, waypoint10.Location, waypoint8.Location, waypoint9.Location }
+Apc2Waypoints = { waypoint8, waypoint7, waypoint6, waypoint5, waypoint4 }
+Apc3Waypoints = { waypoint3, waypoint2, waypoint1, waypoint0, waypoint11, waypoint10, waypoint8, waypoint9 }
+Civ1Waypoints = { waypoint3, waypoint2, waypoint3, waypoint1, waypoint2, waypoint11, waypoint10, waypoint8, waypoint9 }
+Civ2Waypoints = { waypoint3, waypoint2, waypoint1, waypoint11, waypoint10, waypoint8, waypoint9 }
+Hummer1Waypoints = { waypoint8, waypoint7, waypoint6, waypoint5, waypoint4, waypoint3, waypoint2, waypoint1, waypoint0, waypoint11, waypoint10, waypoint8 }
+
+Apc1TriggerFunctionTime = DateTime.Seconds(3)
+
+Apc1TriggerFunction = function()
+	Reinforcements.ReinforceWithTransport(GDI, 'apc', Apc1Units, Apc1Waypoints, nil,
+	function(transport, cargo)
+		Utils.Do(cargo, function(actor)
+			IdleHunt(actor)
+		end)
+	end,
+	nil)
+end
+
+Hum1TriggerFunction = function(actor, discoverer)
+	Convoi.Owner = GDI	
+	MyActors = Utils.Take(2, GDI.GetActorsByType('jeep'))
+	Utils.Do(MyActors, function(actor)
+		Hum1Movement(actor)
+	end)
+end
+
+Civ1Movement = function(unit)
+	Utils.Do(Civ1Waypoints, function(waypoint)
+		unit.AttackMove(waypoint.Location)
+	end)
+end
+
+Civ2Movement = function(unit)
+	Utils.Do(Civ2Waypoints, function(waypoint)
+		unit.AttackMove(waypoint.Location)
+	end)
+end
+
+Apc2Movement = function(unit)
+	Utils.Do(Apc2Waypoints, function(waypoint)
+		unit.AttackMove(waypoint.Location)
+	end)
+	IdleHunt(unit)
+end
+
+Apc3Movement = function(unit)
+	Utils.Do(Apc3Waypoints, function(waypoint)
+		unit.AttackMove(waypoint.Location)
+	end)
+	IdleHunt(unit)
+end
+
+Hum1Movement = function(unit)
+	Utils.Do(Hummer1Waypoints, function(waypoint)
+		unit.AttackMove(waypoint.Location)
+	end)
+	IdleHunt(unit)
+end
+
+Apc2TriggerFunction = function()
+	MyActors = Utils.Take(1, GDI.GetActorsByType('apc'))
+	Utils.Do(MyActors, function(actor)
+		Apc2Movement(actor)
+	end)
+end
+
+WinTriggerFunction = function()
+	Nod.MarkCompletedObjective(NodObjective)
+end
+
+ChangeOwner = function(tran, team)
+	
+	tran.Owner = NodReinforcements
```

Comment:
I assume this workaround is in place for the `if Nod.HasNoRequiredUnits()` check. Just replace it with a bool that is set after the reinforcements are unloaded, reinforce with owner NodReinforcements and keep that while they go away.